conformity/fields/structures.py

```diff
@@ -98,7 +100,12 @@ def __init__(self, _, value):
 
 class Dictionary(Base):
     """
-    A dictionary with types per key (and requirements per key).
+    A dictionary with types per key (and requirements per key). If the `contents` argument is an instance of
+    `OrderedDict`, the field introspection will include a `display_order` list of keys matching the order they exist
```

Comment:
Per Python 3.7 language standard regular `dict` is required to be implemented as ordered.

On the other hand using `OrderedDict` makes clear intent to preserve order, while using regular `dict` does not. So I'm split on which way to go here, but either choice ought to be documented.